tests/sympc/session/session_test.py

```diff
@@ -66,6 +68,20 @@ def test_przs_generate_random_share(get_clients) -> None:
     assert (share.tensor == target_tensor).all()
 
 
+def test_przs_rs_tensor() -> None:
+    """Test przs_generate_random_share method from Session for ReplicatedSharedTensor."""
+    falcon = Falcon(security_type="malicious")
+    session = Session(protocol=falcon)
+    SessionManager.setup_mpc(session)
+    gen1 = get_new_generator(42)
+    gen2 = get_new_generator(43)
+    session.przs_generators = [gen1, gen2]
+    share = session.przs_generate_random_share(shape=(2, 1))
+    assert isinstance(share, ReplicatedSharedTensor)
+    target_tensor = torch.tensor(([-1540733531777602634], [2813554787685566880]))
```

Comment:
Q: Do we need to hard code this value?

Reply:
It was in the previous test for PRZS , I will modify it.:thumbsup: